internal/explore/render.go

```diff
@@ -1541,6 +1541,9 @@ func getLink(s string) string {
 	if mt.IsIndex() {
 		return `https://github.com/opencontainers/image-spec/blob/main/image-index.md`
 	}
+	if strings.HasPrefix(s, "application/vnd.in-toto.") && strings.HasSuffix(s, "+dsse") {
```

Comment:
AFAICT `+dsse` isn't registered as [structured suffix](https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml).

I do see in the link referenced that they suggest its use, but it would be great if they could register it.